topics/home.md

```diff
@@ -15,6 +15,13 @@
 * [New KMM plugin releases](kmm-plugin-releases.md)
 * [What's new in Kotlin for KMM](whats-new-in-kotlin-for-kmm.md)
 * [Multiplatform roadmap](https://youtrack.jetbrains.com/agiles/153-1251/current)
+* [The video "Kotlin Multiplatform Multiverse Episode 1: Meet KMM!" is published](https://www.youtube.com/watch?v=mdN6P6RI__k)
```

Comment:
I'm not sure if it's good to provide the link to the video and to describe and put the same video just below the link.

Reply:
Let's create a separate heading **Videos**. Otherwise, it looks very inconsistent with other items.
And just include the video without such a long description. People don't read anyway if they see pictures and videos. And it looks very inconsistent now. 
Let's try this way.

Reply:
What about leaving at least the heading of the video as I did it now?

Videos
Kotlin Multiplatform Multiverse Episode 1: Meet KMM!
<video ... />

Reply:
Agree. You can also add ` by Ekaterina Petrova, KMM Developer Advocate.` after the title.